lab-brianna/test/tests.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const SLL = require('../lib/sll');
+const expect = require('chai').expect;
+
+//create a list (origList) and append values. Then test that origList is correct. Then test reverse method and compare that origList and new List are not equal to each other.
+describe('test reverse method on single list link', function () {
```

Comment:
**TODO FOR LOST POINTS**

Good start to the test module so far, but you should have three tests for each linked list method.